Analytics/Classes/SEGAnalytics.m

```diff
@@ -438,7 +438,7 @@ + (NSString *)version
 {
     // this has to match the actual version, NOT what's in info.plist
     // because Apple only accepts X.X.X as versions in the review process.
-    return @"3.8.0-beta.1";
+    return @"3.8.0";
```

Comment:
Why3.8?

Reply:
Well, since it was built on top on `3.8.0-beta.1`, but if we are going to make a copy of the repo we can later change this to 0.x or 1.0